src/shilpa/Shop.java

```diff
@@ -0,0 +1,174 @@
+//Assignment  7  : 30th March 2021 
+
+//Create a system (Shop) which maintains each item's quantity.
+
+package shilpa;
+
+public class Shop {
+	int maggieQuantity, dosaQuantity, oilQuantity, panipuriQuantity, masalaQuantity;
+	
+	void Inventory(int maggie,int dosa, int oil,int panipuri,int masala ) {
+		maggieQuantity = maggie;
+		dosaQuantity = dosa;
+		oilQuantity = oil;
+		panipuriQuantity = panipuri;
+		masalaQuantity = masala;	
+	}
+	
+	void customer(int maggieOrder,int dosaOrder, int oilOrder,int panipuriOrder,int masalaOrder) {
+		if(maggieQuantity == 0) 
+		{
+			System.out.println("Maggie Packets running out of stock");
+		}
+		
+		else if(maggieQuantity < maggieOrder)
+		{
+			int remainingmaggy = maggieOrder - maggieQuantity;
+			System.out.println("Sorry we have only "+maggieQuantity+" packets of maggie available, please purchase "+remainingmaggy+" Packets later.");
+			maggieQuantity = 0;
```

Comment:
If request stock is not available mean we are setting direct it zero, 
what if second time your calling same method and let say you request 20 packs, then this looks not logically correct